content/developer/api/Developer-setup-guide/JSON-API.adoc

```diff
@@ -48,7 +48,16 @@ If you would like to change its value you may generate a new one by running
 [source,php]
 php -r 'echo base64_encode(random_bytes(32)), PHP_EOL;'
 
-and then storing the output in the config.php.
+and then storing the output in the Apiconfig.php. Found at `/Api/Core/Config/`.
+
+You should add a new line under :
+[source,php]
+    const OAUTH2_PRIVATE_KEY = 'Api/V8/OAuth2/private.key';
+    const OAUTH2_PUBLIC_KEY = 'Api/V8/OAuth2/public.key';
+
+It should look like the following:
+[source,php]
+const OAUTH2_ENCRYPTION_KEY = '{Encryption Key}';
```

Comment:
This doesn't seem right, we shouldn't be asking people to make core changes in order to get the API working. This value should be coming from the ```oauth2_encryption_key``` in the config. I can't seem to find anywhere that would even use this constant either.